crates/core/src/generator/templater.rs

```diff
@@ -173,11 +173,7 @@ where
         let to = to
             .parse::<Address>()
             .map_err(|_| TemplaterError::ParseAddressFailed(to))?;
-        let value = funcdef
-            .value
-            .as_ref()
-            .map(|s| self.replace_placeholders(s, placeholder_map))
-            .and_then(|s| s.parse::<U256>().ok());
+        let value = funcdef.value;
```

Comment:
this still needs to call `.map(|s| self.replace_placeholders(s, placeholder_map))` -- this change disables placeholder usage in the `value` field